nextflow_schema.json

```diff
@@ -1,3015 +1,2233 @@
+            "type": "string",
+            "description": "Path to Kraken2 database (do not use symlinks)"
+          },
+          "classified_suffix": {
+            "type": "string",
+            "default": "classified",
+            "hidden": true
+          },
+          "unclassified_suffix": {
+            "type": "string",
+            "default": "unclassified",
+            "hidden": true
+          },
+          "report_suffix": {
+            "type": "string",
+            "default": "report",
+            "hidden": true
+          },
+          "output_suffix": {
+            "type": "string",
+            "default": "output",
+            "hidden": true
+          },
+          "save_output_fastqs": {
+            "type": "boolean",
+            "hidden": true
+          },
+          "save_reads_assignments": {
+            "type": "boolean",
+            "default": true,
+            "hidden": true
+          },
+          "run_kraken_quick": {
+            "type": "boolean",
+            "hidden": true
+          },
+          "report_tag": {
+            "type": "string",
+            "default": "KrakenReport",
+            "hidden": true
+          },
+          "tophit_level": {
+            "type": "string",
+            "default": "S",
+            "hidden": true
+          }
+        },
+        "kraken_bin": {
+          "taxonomic_level": {
+            "type": "string",
+            "default": "G",
+            "description": "Taxonomic level to bin contigs at."
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/input_output_options"
+    },
+    {
+      "$ref": "#/definitions/bakta"
+    },
+    {
+      "$ref": "#/definitions/fastp"
+    },
+    {
+      "$ref": "#/definitions/mash"
+    },
+    {
+      "$ref": "#/definitions/quast"
+    },
+    {
+      "$ref": "#/definitions/generic_options"
+    },
+    {
+      "$ref": "#/definitions/skip_options"
+    },
+    {
+      "$ref": "#/definitions/databases_and_pre_computed_files"
+    },
+    {
+      "$ref": "#/definitions/max_job_request_options"
+    },
+    {
+      "$ref": "#/definitions/institutional_config_options"
+    },
+    {
+      "$ref": "#/definitions/seqkit"
+    },
+    {
+      "$ref": "#/definitions/coveragecalculation"
+    },
+    {
+      "$ref": "#/definitions/qcreport"
+    },
+    {
+      "$ref": "#/definitions/seqtk_size"
+    },
+    {
+      "$ref": "#/definitions/medaka"
+    },
+    {
+      "$ref": "#/definitions/staramr"
+    },
+    {
+      "$ref": "#/definitions/mobrecon"
+    },
+    {
+      "$ref": "#/definitions/kat"
+    },
+    {
+      "$ref": "#/definitions/platform_options"
+    },
+    {
+      "$ref": "#/definitions/seqtk"
+    },
+    {
+      "$ref": "#/definitions/flye"
+    },
+    {
+      "$ref": "#/definitions/spades"
+    },
+    {
+      "$ref": "#/definitions/checkm"
+    },
+    {
+      "$ref": "#/definitions/kraken2"
+    }
+  ],
+  "chopper": {
+    "quality": {
```

Comment:
Also, each nested parameter containing any sub parameters needs to be declared `"type": "object"` and have nested parameters under it's own "properties" section.

For example:

```json
"chopper": {
    "type": "object",
    "properties": {
        "quality": {
            "type": "integer",
            "default": 0,
            "hidden": true
        },
        "minlength": {
            "type": "integer",
            "default": 100,
            "hidden": true
       },
       // ...
    }
}
```

Reply:
This should be resolved in the newest upload

Reply:
Thank you. Due to the change to not include nested parameters in the schema, this is also no longer needed.